she_codes_news/news/models.py

```diff
@@ -1,8 +1,24 @@
 from django.db import models
+from django.contrib.auth import get_user_model
 
 
+USER = get_user_model()
+
 class NewsStory(models.Model):
     title = models.CharField(max_length=200)
-    author = models.CharField(max_length=200)
+    # author = models.CharField(max_length=200)
+    author = models.ForeignKey(
+        USER, on_delete = models.CASCADE
+    )
     pub_date = models.DateTimeField()
     content = models.TextField()
+    image = models.TextField(max_length = 3000, blank = True)
```

Comment:
You probably should have used a URLField here, because that would add validation for your url